static/src/javascripts/projects/commercial/modules/prebid/utils.js

```diff
@@ -20,6 +20,14 @@ const currentGeoLocation = once((): string => geolocationGetSync());
 const contains = (sizes: PrebidSize[], size: PrebidSize): boolean =>
     Boolean(sizes.find(s => s[0] === size[0] && s[1] === size[1]));
 
+const stripPrefix = (s: string, prefix: string): string => {
```

Comment:
Why use a Regex here? I reckon either this function takes a regex to make it explicitly a Regex stripper, or just use the StdLib `stripPrefix` function [that's already on strings](https://www.scala-lang.org/api/2.12.3/scala/collection/immutable/StringLike.html#stripPrefix(prefix:String):String)?

Reply:
Oops, JavaScript. This is why we do a review in stead of opening a single comment.

Reply:
This is javascript?